src/containers/Review/ReviewSubmit.tsx

```diff
@@ -21,12 +25,29 @@ type ReviewSubmitProps = {
 
 const ReviewSubmit: React.FC<ReviewSubmitProps> = (props) => {
   const {
-    structure: { thisReview, assignment, canApplicantMakeChanges },
+    structure: { info, thisReview, assignment, canApplicantMakeChanges },
   } = props
+
+  const [fullyAssigned, setFullyAssigned] = useState(false)
+
+  const { data, loading, error } = useGetReviewableQuestionsQuery({
+    variables: {
+      applicationId: info.id,
+      stageId: thisReview?.current.stage.id as number,
+      levelNumber: assignment?.assigneeLevel as number,
+    },
+  })
   const reviewDecision = thisReview?.reviewDecision
   const { decisionOptions, getDecision, setDecision, getAndSetDecisionError, isDecisionError } =
     useGetDecisionOptions(canApplicantMakeChanges, assignment, thisReview)
 
+  useEffect(() => {
+    if (!data) return
+    const { assignedQuestions, reviewableQuestions } = data
+    const fullyAssigned = assignedQuestions?.totalCount === reviewableQuestions?.totalCount
+    setFullyAssigned(fullyAssigned)
+  }, [data])
```

Comment:
I think something like this will work (assuming you remove the "fullyAssigned" state variable above:
```ts
if (!data) return null

const { assignedQuestions, reviewableQuestions } = data
const fullyAssigned = assignedQuestions?.totalCount === reviewableQuestions?.totalCount
```

Without the `useEffect`. There'll be a lot less unnecessary re-renders.